### PR TITLE
[WIP] One import doc to rule them all

### PR DESCRIPTION
**Pull request by rachelwhitton**
Add dash error image, refactor to include export instructions and terminus instructions, remove unnecessary screenshots

Connects to #415 

@nataliejeremy @bmackinney @atjuch @ari-gold - can you review and/or provide feedback?

### CODE REVIEW

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 1]
```diff
@@ -6,53 +6,122 @@ category:
   - getting-started
 keywords: wordpress, importing, import site
 ---
-## Overview  
-<div class="alert alert-info" role="alert"> <strong>Note:</strong> Most WordPress sites with session-using code are relying on PHP's default session manager, which uses temporary files on local disk. Pantheon does not support this because it will not work properly in our distributed environment. You can read <a href="/docs/articles/wordpress/wordpress-and-php-sessions#wordpress-and-php-sessions">more here</a>.</div>
 
-##Prepare for Import  
-You will need a zip or tar.gz archive of your site in three separate files:
+There are three major components that make up a dynamic site:
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: dynamic >> WordPress ?

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 2]
```diff
@@ -6,53 +6,122 @@ category:
...
 
-Popular plugins like [Duplicator](http://wordpress.org/plugins/duplicator/) and [BackupBuddy](http://ithemes.com/codex/page/BackupBuddy) will also do this for you. You should be able to simply upload the archive file they produce and Pantheon will do the rest. For large imports, pasting in a web-readable url (e.g. dropbox secret link) will be much faster than manually uploading. Make sure a link goes directly to the file, not a landing page.
+## Export WordPress via Plugins
+Popular plugins like [Duplicator](/docs/articles/wordpress/clone-a-wordpress-site-with-duplicator-plugin/) and [BackupBuddy](http://ithemes.com/codex/page/BackupBuddy) allow you to create single file archives quickly. For large imports, pasting in a web-readable url (e.g. Dropbox secret link) will be much faster than manually uploading. Make sure a link goes directly to the file, not a landing page.
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: link to our other docs?
https://pantheon.io/docs/articles/wordpress/clone-a-wordpress-site-with-duplicator-plugin/
https://pantheon.io/docs/articles/sites/create/importing-a-large-site/

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 3]
```diff
@@ -6,53 +6,122 @@ category:
...
+├── wp-mail.php
+├── wp-login.php
+├── wp-content
+    ├── index.php
+    ├── mu-plugins
+    ├── themes
+    ├── plugins
+```
+### Create Code Archive
+The code archive should not include the `wp-content/uploads` or any other static assets that should not be tracked in version control. If your codebase contains static files they should be moved to the `wp-content/uploads` directory before export.
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: passive >> active? 
"The code archive should not..."
to
"Do not include the `wp-content/uploads` directory or any other static assets that shouldn't be tracked in Git version control...."

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 4]
```diff
@@ -6,53 +6,122 @@ category:
...
+TARGET=~/Desktop
+SOURCE=~/Projects/mysite
+cd $SOURCE/wp-content/uploads
+tar -czf $TARGET/files.tar.gz .
+```
+
+
+
+## Import Via Terminus
+Using the following [Terminus](https://github.com/pantheon-systems/cli) command, you can import a single file archive in URL format that includes your codebase, database, and files:
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: If I decompress the archive that contains "codebase, database, and files" what will the dir structure look like?

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 5]
```diff
@@ -65,6 +134,13 @@ Currently, we are shipping with one included/recommended plugin, which is design
 
 It sets a default cache lifetime of 10 minutes, and will automatically clear post pages (and taxonomy lists) when new content is created or updated. You can also manually flush the whole cache for a site.
 
+## Troubleshooting
+The following warning can appear on newly created sites where the import of your site archive failed:
+![Unable to Load Git History](/source/docs/assets/images/unable-to-load-git-history.png)
+
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: remove extra line break?

File: source/docs/articles/wordpress/importing-a-wordpress-site.md (hunk 4 above)
Comment (bmackinney) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: Provide a code block dir structure, and a link to an example archive.

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 6]
```diff
@@ -6,53 +6,122 @@ category:
...
+1. **Codebase** - all executable code, plugins, themes, and so forth.
 
-* Database - contains the content of the site and some site configurations.
+2. **Database** - contains the content of the site and some site configurations.
 
-* Files - anything under wp-content/uploads. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
+3. **Files** - anything under `wp-content/uploads`. This houses a combination of uploaded content from site users, along with generated stylesheets, aggregated scripts, image styles, etc.
 
-There are two ways to import. Direct uploads from your desktop can be 100MB max; using a remote URL can be up to 500MB.
+<div class="alert alert-info" role="alert"> <strong>Note:</strong> Most WordPress sites with session-using code are relying on PHP's default session manager, which uses temporary files on local disk. Pantheon does not support this because it will not work properly in our distributed environment. You can read <a href="/docs/articles/wordpress/wordpress-and-php-sessions#wordpress-and-php-sessions">more here</a>.</div>
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: This warning doesn't seem to belong in this article.

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 7]
```diff
@@ -6,53 +6,122 @@ category:
...
+```php
+TARGET=~/Desktop
+SOURCE=~/Projects/mysite
+cd $SOURCE/wp-content/uploads
+tar -czf $TARGET/files.tar.gz .
+```
+
+
+
+## Import Via Terminus
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: I think this should move below "Import via Dashboard" section

File: source/docs/articles/wordpress/importing-a-wordpress-site.md [hunk 8]
```diff
@@ -6,53 +6,122 @@ category:
...
+<div class="alert alert-info" role="alert">
+<strong>Note</strong>: Direct uploads from your desktop can be 100MB max; using a remote URL can be up to 500MB.</div>
+
 ![](/source/docs/assets/images/desk_images/259156.png)  
-Alternatively, you can enter the URL to your site code (required), user files (optional) and database (optional).
+Alternatively, you can provide separate archives for code (required), user files (optional) and database (optional).
 ![](/source/docs/assets/images/desk_images/247522.png)
 
-## Relax While We Configure Your Codebase
+#### Relax While We Configure Your Codebase
```
Comment (ari-gold) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: Configure Your Codebase >> Import Your Site    ?

**Comment by ari-gold** [comment]
Much better! +1

File: source/docs/articles/wordpress/importing-a-wordpress-site.md (hunk 4 above)
Comment (rachelwhitton) [inline source/docs/articles/wordpress/importing-a-wordpress-site.md]: @bmackinney what kind of example archive do you want to see - blank WP site created on an alternate host? How should we link to that example other than providing a code block of the dir structure?

**Comment by rachelwhitton** [comment]
@bmackinney -  This PR now: 
- Creates CMS specific docs for creating site archives
 - [Export an Existing Drupal Site](https://github.com/pantheon-systems/documentation/blob/importing-wp-terminus-clean-up/source/docs/articles/drupal/prepare-drupal-for-export.md)
 - [Export an Existing WordPress Site](https://github.com/pantheon-systems/documentation/blob/importing-wp-terminus-clean-up/source/docs/articles/wordpress/export-an-existing-wordpress-site.md)
- Creates a single import doc that cross links between both CMS export docs
- Deletes existing WordPress import doc (Replaced with WordPress export doc that now links to a single import doc)
- Deletes "migrating-sites.md" as it is duplicate content. 

Can you review and provide feedback/guidance for issue #415 ? Is this more along the lines of what you're looking for?  


Redirects needed upon deployment:
1. [Migrate Sites from Other Hosts](https://pantheon.io/docs/articles/sites/create/migrating-sites/) redirects to [Importing an Existing Site](https://pantheon.io/docs/articles/sites/create/importing-an-existing-site/)
2. [Importing a WordPress Site](https://pantheon.io/docs/articles/wordpress/importing-a-wordpress-site/) redirects to [Export an Existing WordPress Site (not live yet)](https://pantheon.io/docs/articles/wordpress/export-an-existing-wordpress-site)

**Comment by rachelwhitton** [comment]
This work is being carried out in PR #725

**Pull request closed without merging**